src/gui/tray/Window.qml

```diff
@@ -685,28 +685,11 @@ Window {
             text: UserModel.currentUser.unifiedSearchResultsListModel.searchTerm
 
             property bool isSearchRunning: UserModel.currentUser.unifiedSearchResultsListModel.isSearchInProgress
+            property bool waitingForSearchTermEditEnd: UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd

```

Comment:
I am curious why you do not directly use `UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd` in line `692`

Reply:
It's mainly for readability, `UserModel.currentUser.unifiedSearchResultsListModel.waitingForSearchTermEditEnd` is very long